apps/sms/views/shared/js/navigation.js

```diff
@@ -751,6 +751,13 @@ var Navigation = {
       return Promise.resolve();
     }
 
+    if (navigationTransition) {
```

Comment:
I agree it's more difficult to move the isCurrentPanel to startNavigation, so why not.

Also we already have https://github.com/steveck-chung/gaia/blob/33818ef40c431f07da32b968aa7788ef5340180e/apps/sms/views/shared/js/navigation.js#L322-L327 but I think we need to keep it too, because we can access startNavigation from other places (like "back" or "hashchange" event).